app/services/laa_reference_unlinker.rb

```diff
@@ -6,7 +6,12 @@ def initialize(defendant_id:, user_name:, unlink_reason_code:, unlink_other_reas
     @user_name = user_name
     @unlink_reason_code = unlink_reason_code
     @unlink_other_reason_text = unlink_other_reason_text
-    @laa_reference = LaaReference.find_by!(defendant_id: defendant_id, linked: true)
+
+    @laa_reference = LaaReference.find_by(defendant_id: defendant_id)
+
+    unless laa_reference.linked
+      Rails.logger.warn("LaaReferenceUnlinker: the defendant '#{defendant_id}' is already unlinked.")
```

Comment:
Hi, just to confirm that this will not be sent back to CDUI? The unlinked error

Reply:
that's right, it is only logged to Kibana.